get-iap-token/src/main.ts

```diff
@@ -0,0 +1,24 @@
+import * as core from '@actions/core';
+import { GoogleToken } from 'gtoken';
+import { getCredentials } from './credentials';
+
+// Retrieve input.
```

Comment:
Is there a reason why the the operation is not async?
https://github.com/actions/typescript-action#change-the-code